templates/ts-empty/package.json

```diff
@@ -11,7 +11,11 @@
         "crawlee": "^3.5.4"
     },
     "devDependencies": {
+        "@apify/eslint-config-ts": "^0.3.0",
         "@apify/tsconfig": "^0.1.0",
+        "@typescript-eslint/eslint-plugin": "^6.7.2",
+        "@typescript-eslint/parser": "^6.7.2",
```

Comment:
why not v7?

Reply:
I found this version in other templates, and I don't know where to find the actual used version across projects (the latest version for @typescript-eslint/parser is 8.5)

Reply:
> the latest version for @typescript-eslint/parser is 8.5

yes, and latest version of eslint is v9, we cant use those now since they require the new flat configs. but we can use v7 just fine.

Reply:
Should I change it and make a new PR from the branch from the upstream repo?
Check if I understand you correctly:
-> "@typescript-eslint/eslint-plugin": "^7.18.0"
-> "@typescript-eslint/parser": "^7.18.0"

Reply:
yes please, otherwise the tests wont run. latest v7 should be fine, we use that in all our repos afaik.